xbmc/linux/LinuxTimezone.h

```diff
@@ -59,6 +60,7 @@ class CLinuxTimezone : public ISettingCallback, public ISettingsHandler
    std::map<std::string, std::string> m_countriesByTimezoneName;
 };
 
-extern CLinuxTimezone g_timezone;
+XBMC_GLOBAL_REF(CLinuxTimezone,g_timezone);
+#define g_timezone XBMC_GLOBAL_USE(CLinuxTimezone)
```

Comment:
In order to have control every object needs an owner. There is an ongoing process to kill this globals and singletons one by one and we don't open the backdoor here.